ann_cnn_training/training.py

```diff
@@ -30,20 +30,20 @@
 torch.set_printoptions(edgeitems=2)
 torch.manual_seed(123)
 
-f = "/home/users/ag4680/myjupyter/137levs_ak_bk.npz"
-data = np.load(f, mmap_mode="r")
+# f = "/home/users/ag4680/myjupyter/137levs_ak_bk.npz"
+# data = np.load(f, mmap_mode="r")
 
-lev = data["lev"]
-ht = data["ht"]
-ak = data["ak"]
-bk = data["bk"]
-R = 287.05
-T = 250
-rho = 100 * lev / (R * T)
+# lev = data["lev"]
+# ht = data["ht"]
+# ak = data["ak"]
+# bk = data["bk"]
+# R = 287.05
+# T = 250
+# rho = 100 * lev / (R * T)
```

Comment:
As a general rule, I think it is better to remove code if it is no longer needed rather than comment it out.

Comments should be used to describe code, not for keeping potentially useful snippets.

If you do want to keep it, consider refactoring it, so it can be enabled/disabled with a command line argument or environment variable.